coral/src/app/pages/not-found/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Typography } from "@aivenio/aquarium";
+import AuthenticationRequiredBoundary from "src/app/components/AuthenticationRequiredBoundary";
+import Layout from "src/app/layout/Layout";
+
+const NotFound = () => {
+  return (
+    <AuthenticationRequiredBoundary>
+      <Layout>
+        <Typography.Heading color={"secondary-100"}>
+          Page not found
+        </Typography.Heading>
+
+        <Typography.LargeText>
```

Comment:
I am not completely sure how casual our tone of voice should be. I find this a bit on the more casual side, so I would remove it unless we explicitly want to be a bit "playful"

Reply:
Yes, I planned on tagging Harshini on the copy before opening it to review. This was for demo purposes ^^

Reply:
Edited: [bba44e0](https://github.com/aiven/klaw/pull/358/commits/bba44e08c4d1fcc051a93e99fecf724e410931bb)